public/admin/config.yml

```diff
@@ -0,0 +1,38 @@
+backend:
```

Comment:
Consider adding a comment at the start linking to the docs, and maybe explaining that the branch option is where the copy updates will go